homework_2/task4.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#define len 15
+
+void sort(int ar[], int ln)
+{
+	int point = ar[0];
+	int left = 0;
+	int right = ln - 1;
+	do
+	{
+		while (ar[left] < point)
+		{
+			left++;
+		}
+		while (ar[right] > point)
+		{
+			right--;
+		}
+		if (left <= right)
+		{
+			int temp = ar[left];
+			ar[left] = ar[right];
+			ar[right] = temp;
+			left++;
+			right--;
+		}
+	} while (left <= right);
+
+}
+int main()
+{
+	int arr[len] = { 0 };
```

Comment:
```suggestion
      	srand(time(0));
	int arr[len] = { 0 };
```
Чтобы каждый раз псевдослучайные числа разными были